mediator/esmFldsExchange_ufs_mod.F90

```diff
@@ -716,15 +752,13 @@ subroutine esmFldsExchange_ufs(gcomp, phase, rc)
     if ( trim(coupling_mode) == 'ufs.nfrac.aoflux') then
        allocate(flds(21))
        flds = (/'Sa_z      ', 'Sa_topo   ', 'Sa_tbot   ', 'Sa_pbot   ', &
-                'Sa_shum   ', 'Sa_u      ', 'Sa_v      ', 'Faxa_lwdn ', &
-                'Sa_ptem   ', 'Sa_dens   ', 'Faxa_swdn ', 'Sa_pslv   ', &
-                'Faxa_snowc', 'Faxa_snowl', 'Faxa_rainc', 'Faxa_rainl', &
-                'Faxa_swndr', 'Faxa_swndf', 'Faxa_swvdr', 'Faxa_swvdf', &
-                'Faxa_swnet'/)
+                'Sa_shum   ', 'Sa_u      ', 'Sa_v      ', 'Sa_pslv   ', &
+                'Faxa_lwdn ', 'Faxa_swdn ', 'Faxa_snowc', 'Faxa_snowl', &
+                'Faxa_rainc', 'Faxa_rainl', 'Faxa_rain ', 'Faxa_swnet'/)
     else
        allocate(flds(18))
        flds = (/'Sa_z      ', 'Sa_ta     ', 'Sa_pslv   ', 'Sa_qa     ', &
-                'Sa_ua     ', 'Sa_va     ', 'Faxa_swdn ', 'Faxa_lwdn ', &
+                'Sa_u      ', 'Sa_v      ', 'Faxa_swdn ', 'Faxa_lwdn ', &
                 'Faxa_swnet', 'Faxa_rain ', 'Sa_prsl   ', 'vfrac     ', &
                 'Faxa_snow ', 'Faxa_rainc', 'Sa_tskn   ', 'Sa_exner  ', &
```

Comment:
Can we make an alias is fd_ufs.yaml for vfrac and zorl to be ``Sa_frac`` and ``Sa_zorl`` and use those names in the flds array? That keeps it consistent w/ all the other field names.

Reply:
@DeniseWorthen The vfrac is also used by JEDI interface. I'll try to make those changes but i am not sure how I could test JEDI interface and be sure my changes are not breaking anything in that side. Let me know what you think?

Reply:
Shouldn't it be fine if you just alias vfrac with ``Sa_vfrac`` and then use ``Sa_vfrac`` here (same for zorl)?

Reply:
@DeniseWorthen Yes, probably it would be fine.